app/models/like.rb

```diff
@@ -2,9 +2,9 @@ class Like < ApplicationRecord
   belongs_to :author, class_name: 'User'
   belongs_to :post, counter_cache: :likes_counter
```

Comment:
- [ ] **[OPTIONAL]** When I like a post it increments the `likes_counter` by 2 instead of 1. Kindly inspect that and resolve it. I think it has to do with this `counter_cache`. When I removed it everything works just fine.

- [ ] **[OPTIONAL]** The same issue is present in your comment model. Adding a comment increments the post's `comments_counter` by 2, instead of 1. Kindly look into that as well.